### ISSUE TITLE
Unable to get provider androidx.startup.InitializationProvider:  cannot locate symbol "__emutls_get_address"

### ISSUE THREAD

**Issue by apaolino** [isar/isar#576]
**Steps to Reproduce**
Thanks for this project!
Since today's 3.0.0-dev.9 release I'm not able to start my app anymore. Having this crash:
```
E/AndroidRuntime(11096): java.lang.RuntimeException: Unable to get provider androidx.startup.InitializationProvider: androidx.startup.StartupException: java.lang.UnsatisfiedLinkError: dlopen failed: cannot locate symbol "__emutls_get_address" referenced by "/data/app/~~z6HHjsme2GLLxSB_ujdm8Q==/com.offertadelgiorno.offertadelgiorno.dev-2W6Pu0v3XjnGLeurtx_vpg==/base.apk!/lib/arm64-v8a/libisar.so"...
E/AndroidRuntime(11096): 	at android.app.ActivityThread.installProvider(ActivityThread.java:7436)
E/AndroidRuntime(11096): 	at android.app.ActivityThread.installContentProviders(ActivityThread.java:6948)
E/AndroidRuntime(11096): 	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6719)
E/AndroidRuntime(11096): 	at android.app.ActivityThread.access$1500(ActivityThread.java:256)
E/AndroidRuntime(11096): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2090)
E/AndroidRuntime(11096): 	at android.os.Handler.dispatchMessage(Handler.java:106)
E/AndroidRuntime(11096): 	at android.os.Looper.loopOnce(Looper.java:201)
E/AndroidRuntime(11096): 	at android.os.Looper.loop(Looper.java:288)
E/AndroidRuntime(11096): 	at android.app.ActivityThread.main(ActivityThread.java:7842)
E/AndroidRuntime(11096): 	at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime(11096): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
E/AndroidRuntime(11096): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
```

**Version**
 - Platform: Android
 - Flutter version:  3.0.4 
 - Isar version: 3.0.0-dev.9
 - Isar Generator version: 3.0.0-dev.9

**Comment by leisim** [comment]
Oh! Thanks for opening this issue. I'll take care of this.

**Comment by apaolino** [comment]
I've run it only on emulator as of now, API level 32

**Comment by leisim** [comment]
Fixed in `3.0.0-dev.10`

**Comment by stefan-weinand** [comment]
Thanks for asking and for fixing. I searched hours without success on this problem.
